contracts/tokenbridge/ethereum/L1AtomicTokenBridgeCreator.sol

```diff
@@ -85,26 +106,19 @@ contract L1AtomicTokenBridgeCreator is Initializable, OwnableUpgradeable {
         _disableInitializers();
     }
 
-    function initialize() public initializer {
+    function initialize(L1TokenBridgeRetryableSender _retryableSender) public initializer {
```

Comment:
Why change this to being passed in? It's possible for the initialize to be frontrun with this now right? Not a security problem, but could be annoying.

Reply:
Contract size limit, retryableSender's bytecode doesn't fit anymore. Here's current breakdown:
```
❯ forge build --sizes | egrep "L1AtomicTokenBridgeCreator|L1TokenBridgeRetryableSender|L2Atomic"
| L1AtomicTokenBridgeCreator      | 24.211    | 0.365       |
| L1TokenBridgeRetryableSender    | 2.743     | 21.833      |
| L2AtomicTokenBridgeFactory      | 23.329    | 1.247       |
```

Reply:
kk, we should maybe add a typescript check that the correct retryableSender was set?

Reply:
Added - https://github.com/OffchainLabs/token-bridge-contracts/pull/37/commits/6cb20600a3f964aa517ea024fc22346e3a0dd16d